confd/vars/main.yaml

```diff
@@ -161,6 +161,8 @@ kilda_storm_parallelism_level: 1
 kilda_storm_flow_hs_parallelism: 10
 kilda_storm_parallelism_workers_count: 1
 
+kilda_storm_flow_monitoring_parallelism: 1
```

Comment:
is it enough?

Reply:
Current flow and isl caches are implemented without parallelism support.